File: python/easy/stand_in_line.py
# Stand in Line
# https://edabit.com/challenge/S26tvW7BPrJsyJApt

def next_in_line(lst, num):
	if not lst:
		return "No list has been selected"
	else:
		lst.pop(0)
		lst.append(num)
	
	return lst

File: javascript/easy/stand_in_line.js
// Stand in Line
// https://edabit.com/challenge/ebm9Yo3nxxniskr64

function nextInLine(arr, num) {
    if (arr) {
        arr.push(num);
        arr.shift();
        return arr;
    } else {
        return "No array has been selected";
    }
}